specification/maps/resource-manager/Microsoft.Maps/stable/2021-02-01/maps-management.json

```diff
@@ -0,0 +1,1105 @@
+          "$ref": "#/definitions/Kind"
+        },
+        "systemData": {
+          "readOnly": true,
+          "description": "The system meta data relating to this resource.",
+          "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/systemData"
+        },
+        "properties": {
+          "description": "The map account properties.",
+          "$ref": "#/definitions/MapsAccountProperties"
+        }
+      },
+      "required": [
+        "location",
+        "sku"
+      ],
+      "allOf": [
+        {
+          "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/TrackedResource"
+        }
+      ],
+      "x-ms-azure-resource": true
+    },
+    "MapsAccountUpdateParameters": {
+      "description": "Parameters used to update an existing Maps Account.",
+      "type": "object",
+      "properties": {
+        "tags": {
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          },
+          "description": "Gets or sets a list of key value pairs that describe the resource. These tags can be used in viewing and grouping this resource (across resource groups). A maximum of 15 tags can be provided for a resource. Each tag must have a key no greater than 128 characters and value no greater than 256 characters."
+        },
+        "kind": {
+          "description": "Get or Set Kind property.",
+          "$ref": "#/definitions/Kind"
+        },
+        "sku": {
+          "description": "The SKU of this account.",
+          "$ref": "#/definitions/Sku"
+        },
+        "properties": {
+          "description": "The map account properties.",
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/MapsAccountProperties"
+        }
+      }
+    },
+    "CreatorUpdateParameters": {
+      "description": "Parameters used to update an existing Creator resource.",
+      "type": "object",
+      "properties": {
+        "tags": {
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          },
+          "description": "Gets or sets a list of key value pairs that describe the resource. These tags can be used in viewing and grouping this resource (across resource groups). A maximum of 15 tags can be provided for a resource. Each tag must have a key no greater than 128 characters and value no greater than 256 characters."
+        },
+        "properties": {
+          "description": "Creator resource properties.",
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/CreatorProperties"
+        }
+      }
+    },
+    "MapsAccounts": {
+      "description": "A list of Maps Accounts.",
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "a Maps Account.",
+          "type": "array",
+          "readOnly": true,
+          "items": {
+            "$ref": "#/definitions/MapsAccount"
+          }
```

Comment:
as mentioned above too - adding a nextLink here is good practice even if you don't need it right now

Reply:
added name, but the value from the RP will be null